src/views/con-beheer-views/con-beheer-views.js

```diff
@@ -650,10 +634,52 @@ const ConBeheerViews = ({ store }) => {
                     </ConActionMenu.Button>
                   </ConActionMenu.Menu>
                 </ConActionMenu>
-              )}
-            </div>
+              </div>
+            )}
           </div>
 
+          {/* Filters Section */}
+          {gemma?.get_view && !gemma?.get_viewError && (
+            <div className='con-beheer-views-filters-section'>
+              <div className='con-beheer-views-filters-header'>
+                <h3>Weergave filters</h3>
+                <p>
+                  Pas de weergave aan door specifieke elementen te tonen of te
+                  verbergen
+                </p>
+              </div>
+
+              <div className='con-beheer-views-filters-flex'>
+                <AcCheckbox
+                  label='Gebruik'
+                  checked={filters.gebruik}
+                  tooltip={
+                    'Toon elementen gerelateerd aan het gebruik van diensten en applicaties'
+                  }
+                  onChange={(checked) => handleToggleFilter('gebruik')(checked)}
+                />
+
+                <AcCheckbox
+                  label='Product'
+                  checked={filters.product}
+                  tooltip={
+                    'Toon product-gerelateerde elementen en hun onderlinge relaties'
+                  }
+                  onChange={(checked) => handleToggleFilter('product')(checked)}
+                />
+
+                <AcCheckbox
+                  label='Deelnames'
+                  checked={filters.deelnames}
+                  tooltip={
+                    'Toon deelname-gerelateerde elementen en participatie-aspecten'
+                  }
+                  onChange={(checked) => handleToggleFilter('deelnames')(checked)}
```

Comment:
You can actually remove the arrow function for this.

```suggestion
                  onChange={handleToggleFilter('gebruik')}
                />

                <AcCheckbox
                  label='Product'
                  checked={filters.product}
                  tooltip={
                    'Toon product-gerelateerde elementen en hun onderlinge relaties'
                  }
                  onChange={handleToggleFilter('product')}
                />

                <AcCheckbox
                  label='Deelnames'
                  checked={filters.deelnames}
                  tooltip={
                    'Toon deelname-gerelateerde elementen en participatie-aspecten'
                  }
                  onChange={handleToggleFilter('deelnames')}
```